content/5_Plat/HLD.mdx

```diff
@@ -252,7 +252,325 @@ set problems where $\Theta(Q\sqrt N\log N)$ is intended ...
 	</Resource>
 </Resources>
 
-<IncompleteSection />
+
+### Implementation 1 - Basic Heavy-Light Decomposition
```

Comment:
could you add a focus problem instead of just an implementation?

sorry if this wasn't clear from the bounty description

Reply:
done